docs/tutorials/api/compute.rst

```diff
@@ -268,293 +158,68 @@ and the function automatically runs on your specified hardware!
 
 
 .. parsed-literal::
-    :class: code-output
-
-    INFO | 2023-08-29 03:59:43.821391 | Calling getpid.call
-
-
-.. parsed-literal::
-    :class: code-output
-
-    local function result: 7592
-    base servlet: Calling method call on module getpid
-
-
-.. parsed-literal::
-    :class: code-output
-
-    INFO | 2023-08-29 03:59:44.078775 | Time to call getpid.call: 0.26 seconds
-
-
-.. parsed-literal::
-    :class: code-output
-
-    remote function result: 1382396
-
-
-Git Functions
-~~~~~~~~~~~~~
-
-A neat feature of Runhouse is the ability to take a function from a
-Github repo, and create a wrapper around that function to be run on
-remote. This saves you the effort of needing to clone or copy a
-function. To do so, simply pass in the function url into
-``rh.function``.
-
-We’ve implemented the same ``getpid`` function
-`here <https://github.com/run-house/runhouse/blob/main/docs/notebooks/sample_fn.py>`__.
-Below, we demonstrate how we can directly use the GitHub link and
-function name to run this function on remote hardware, without needing
-to clone the repo ourselves or reimplement the function locally.
-
-.. code:: ipython3
-
-    pid_git_remote = rh.function(
-        fn='https://github.com/run-house/runhouse/blob/main/docs/notebooks/sample_fn.py:getpid',
-        system=cluster,
-    )
-
-
-.. parsed-literal::
-    :class: code-output
-
-    INFO | 2023-08-29 04:00:01.870718 | Setting up Function on cluster.
-    INFO | 2023-08-29 04:00:01.873021 | Copying package from file:///Users/caroline/Documents/runhouse/runhouse to: cpu-cluster
-    INFO | 2023-08-29 04:00:03.145979 | Copying package from file:///Users/caroline/Documents/runhouse/runhouse to: cpu-cluster
-    INFO | 2023-08-29 04:00:04.625905 | Calling env_20230829_035957.install
-
-
-.. parsed-literal::
-    :class: code-output
-
-    base servlet: Calling method install on module env_20230829_035957
```

Comment:
I cleared the output for a bunch of these longer log outputs. I think we also need to revisit which ones we set to be logger.info vs debug in terms of what processes we want to be logged during processes. and possibly in tutorials keep it at the info/error level for cleanliness if we want to show output.

Reply:
Oh I thought I cleared all output before I pushed. Yes, makes sense. 

Reply:
yup you did clear them -- I generally like to leave them in to display the output of the code (for users that might just want to skim through without setting up yet), but removed the more cluttery setup/logs output